src/popup/pages/SignTransaction.vue

```diff
@@ -51,8 +58,24 @@ export default defineComponent({
           return;
         }
 
+        const replaceCallerTx = async (tx: Encoded.Transaction) => {
```

Comment:
1. I strongly suggest to add comment that would describe: the reason for replacing caller tx, way of testing this, why/how/when needs this. 
2. Within the components we define functions with the `function functionName() {}` notation to make it easier to distinguish constants from functions.

Reply:
agree, will adjust it, thanks!

Reply:
adjusted

Reply:
@thepiwo you didn't add the comment.

Reply:
my bad, will do

Reply:
done